frontend/ui/data/keyboardlayouts/el_keyboard.lua

```diff
@@ -75,17 +75,17 @@ return {
             { _T_el,  _t_el,  "+",    _eq,    "À",    "ὒ",    "5",    "¼", },
             { _Y_el,  _y_el,  "€",    "(",    "Ύ",    "ύ",    "6",    "½", },
             { _U_el,  _u_el,  "‰",    ")",    "Æ",    "ὓ",    "7",    "¾", },
-            { _I_el,  _i_el,  "|",    "\\",   "Ί",    "ί",    "8",    "©", },
+            { _I_el,  _i_el,  "|",    "\",   "Ί",    "ί",    "8",    "©", },
```

Comment:
Are you sure about this change ? a double \ is needed (the first to "escape" the 2nd, so the 2nd means a \ - otherwise, a single \ is escaping what comes next, so \" is making the closing " not close :)
I'm pretty sure this will crash, so please check on your device.

Reply:
Yes, @poire-z is correct - `\` will crash, while `\\` outputs a proper backslash.

Reply:
Oh! I'm sorry. I was wondering why there is a double \ while a single one is displayed on the keyboard.
Keep it as it was.